content/800-data-platform/250-environments/index.mdx

```diff
@@ -23,7 +23,7 @@ For each environment in a [project](/data-platform/projects), you define:
 
 ## Default environment
 
-When you first [add a project](/data-platform/projects/add-project), you define the default environment of the project. The default environment has the name **Production**.
+When you first [create a project](/data-platform/projects/create), you define the default environment of the project. The default environment has the name **Production**.
```

Comment:
Uses actually do not define the default environment. We create it for them and name it to be `Production`